overlord/assertstate/assertstate.go

```diff
@@ -55,8 +55,16 @@ func findError(format string, ref *asserts.Ref, err error) error {
 	}
 }
 
+type RefreshDeclarationOptions struct {
```

Comment:
this should follow the function name and have a plural:

RefreshSnapDeclarationsOptions